spec/pluralrules.html

```diff
@@ -0,0 +1,360 @@
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-pluralrules-constructor">
+    <h1>Properties of the Intl.PluralRules Constructor</h1>
+
+    <p>
+      The Intl.PluralRules constructor has the following properties:
+    </p>
+
+    <emu-clause id="sec-intl.pluralrules.prototype">
+      <h1>Intl.PluralRules.prototype</h1>
+
+      <p>
+        The value of `Intl.PluralRules.prototype` is %PluralRulesPrototype%.
+      </p>
+      <p>
+        This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-intl.pluralrules.supportedlocalesof">
+      <h1>Intl.PluralRules.supportedLocalesOf ( _locales_ [, _options_ ] )</h1>
+
+      <p>
+        When the `supportedLocalesOf` method is called with arguments _locales_ and _options_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Let _availableLocales_ be %PluralRules%.[[AvailableLocales]].
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. Return ? SupportedLocales(_availableLocales_, _requestedLocales_, _options_).
+      </emu-alg>
+
+      <p>
+        The value of the `length` property of the *supportedLocalesOf* method is 1.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-intl.pluralrules-internal-slots">
+      <h1>Internal slots</h1>
+
+      <p>
+        The value of the [[AvailableLocales]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref>.
+      </p>
+
+      <p>
+        The value of the [[RelevantExtensionKeys]] internal slot is *[]*.
+      </p>
+
+      <emu-note>
+        Unicode Technical Standard 35 describes no locale extension keys that are relevant to pluralization process.
+      </emu-note>
+
+      <emu-note>
+        It is recommended that implementations use the locale data provided by the Common Locale Data Repository (available at <a href="http://cldr.unicode.org/">http://cldr.unicode.org/</a>).
+      </emu-note>
+
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-properties-of-intl-pluralrules-prototype-object">
+    <h1>Properties of the Intl.PluralRules Prototype Object</h1>
+
+    <p>
+      The Intl.PluralRules prototype object is itself an ordinary object. <dfn>%PluralRulesPrototype%</dfn> is not an Intl.PluralRules instance and does not have an [[InitializedPluralRules]] internal slot or any of the other internal slots of Intl.PluralRules instance objects.
+    </p>
+    <p>
+      In the following descriptions of functions that are properties or [[Get]] attributes of properties of %PluralRulesPrototype%, the phrase "this PluralRules object" refers to the object that is the this  value for the invocation of the function; a *TypeError* exception is thrown if the this value is not an object or an object that does not have an [[InitializedPluralRules]] internal slot.
```

Comment:
The phrase "this PluralRules object" is unused, so we can remove this paragraph.